test/UrlHelperTest.php

```diff
@@ -48,7 +47,7 @@ public function testRaisesExceptionOnInvocationIfNoRouteProvidedAndResultIndicat
 
     public function testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided()
     {
-        $this->router->generateUri('foo', [])->willThrow(RouterException::class);
+        $this->router->generateUri('foo', [], [])->willThrow(RouterException::class);
```

Comment:
The helper will now always pass [`$routerOptions`](https://github.com/zendframework/zend-expressive-helpers/pull/33/files#diff-c051d24dcf87b2f1bfb6a96ed36220a3R82) to the `RouterInterface::generateUri(...)` method. `$routerOptions` defaults to `[]` if no `$options['router']` value is present, so the test needs to expect an empty `[]` as the 3rd parameter now.

**Note:** Yes, we do have a [test](https://github.com/zendframework/zend-expressive-helpers/pull/33/files#diff-22ee52d336c85d9669c2686e7bab73b4R260) for the "`$options['router'] _not_ being empty" case.